portfolio.js

```diff
@@ -55,8 +55,238 @@ links.forEach((link) => {
+    image: 'images/component_5.png',
+    imageDesktop: 'images/component_5.png',
+    alt: 'Project number five',
+    keywords: [
+      'HTML/CSS',
+      'Ruby on Rails',
+      'Javascript'],
+    liveVersion: '#',
+    source: '#',
+
+  },
+  {
+    id: 6,
+    class: 'six',
+    classDesktop: 'six-desktop',
+    classImgDesktop: 'six-desktop',
+    boxClass: 'boxDsk6',
+    name: 'Project name goes here',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nis. Ut aliquip ex ea commodo consequat.  </br> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    image: 'images/component_6.png',
+    imageDesktop: 'images/component_6.png',
+    alt: 'Project number six',
+    keywords: [
+      'HTML/CSS',
+      'Ruby on Rails',
+      'Javascript'],
+    liveVersion: '#',
+    source: '#',
+
+  },
+];
+
+function generateProjectCards(projects) {
+  const projectCards = [];
+  for (let i = 0; i < projects.length; i += 1) {
+    const project = projects[i];
+    const projectCard = `
+      <div class="box ${project.class}">
+        <img class="component" src="${project.image}" alt="${project.alt}">
+        <img class="${project.classImgDesktop}" src="${project.imageDesktop}" alt="${project.alt}">
+        <h3 class="first-title">${project.name}</h3>
+        <ul class="keywords">
+          ${project.keywords.map((keyword) => `<li>${keyword}</li>`).join('')}
+        </ul>
+        <div>
+          <button class="buttonbox" onclick="showModal(${project.id})">See this project 🡲</button>
+        </div>
+      </div>
+      <div class="${project.classDesktop}">
+        <img class="" src="${project.imageDesktop}" alt="${project.alt}">
+        <div class="${project.boxClass}">
+          <h3 class="first-title">${project.name}</h3>
+          <ul class="keywords">
+            ${project.keywords.map((keyword) => `<li>${keyword}</li>`).join('')}
+          </ul>
+          <div class="buttonboxDesktop" >
+            <button onclick="showModal(${project.id})">See this project 🡲</button>
+          </div>
+        </div>
+      </div>
+      <div class="geometry-center"> <img src="images/image-geometry_2.png" alt=""> </div>
+      <div class="pc-right-squares"> <img src="images/image-geometry_3.png" alt=""> </div>
+      <div class="dot-right-yellow"> <img src="images/image-geometry_4.png" alt=""> </div>
+      <div class="pc-left-squares"> <img src="images/image-geometry_5.png" alt=""> </div>
+    `;
+    projectCards.push(projectCard);
+  }
```

Comment:
- [ ] You have the same issue with the mobile version please see the responsiveness on the mobile version.
- [ ] [Optional]  I noticed after the modal pop-up has been displayed, then when you scroll down, the main page start showing, and the pop-up will be scrolled up. Kindly use fixed positioning to change this strange functionality. 
### you design
![mobile](https://user-images.githubusercontent.com/81727988/211178118-2ce787bf-486c-49b3-a2f9-816f04ddc502.png)